src/one_dragon_qt/view/one_dragon/one_dragon_run_interface.py

```
@@ -230,28 +228,27 @@ def show_notify_dialog(self) -> None:
         dialog = NotifyDialog(self, self.ctx)
         dialog.exec()
 
-    def get_setting_dialog_map(self) -> dict[str, Callable]:
-        """返回 app_id -> 设置弹窗回调 的映射，由子类实现"""
-        return {}
-
     def on_app_setting_clicked(self, app_id: str) -> None:
-        """处理应用设置按钮被点击，查找 get_setting_dialog_map 中的回调并调用"""
-        dialog_fn = self.get_setting_dialog_map().get(app_id)
-        if dialog_fn is None:
+        """处理应用设置按钮被点击，委托给 app_setting_manager"""
```

Ruff (0.15.6)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)